docs/reference/commands/keystore.asciidoc

```diff
@@ -70,12 +71,14 @@ protected, you are prompted to enter the current password and the new one. You
 can optionally use an empty string to remove the password. If the keystore is
 not password protected, you can use this command to set a password.
 
-`remove <setting>`:: Removes a setting from the keystore.
+`remove <settings>`:: Removes settings from the keystore. Multiple setting
+names can be specified as arguments to the `add` command.
 
 `-s, --silent`:: Shows minimal output.
 
-`--stdin`:: When used with the `add` parameter, you can pass the setting value
-through standard input (stdin). See <<add-string-to-keystore>>.
+`--stdin`:: When used with the `add` parameter, you can pass the settings values
```

Comment:
Should we document `-x` too?

Reply:
I notice we only have `-f` without `--force` as well.

Reply:
Thanks for noticing these. Since these are orthogonal to this change, I will handle them in a follow up.

Reply:
I opened #54242.